careers/software-engineer.md

```diff
@@ -4,52 +4,56 @@ The goal of the Software Engineer path is to train engineers to master the funda
 
-## Engineer II
+**Collaboration:** They pair frequently. They are able to quickly recognize when they are stuck and ask for help. For more complicated tasks, they leverage the knowledge of their peers to plan and execute their work on time. They actively participate in code review and offer feedback to others.
 
-**Technical Ability:** Engineer IIs display solid understanding of core engineering concepts. They are focused on growing as an engineer, learning the team's tools and current processes, and developing productivity skills, as well as a strong background in Engineering best-practices like testing, source control, and agile planning. They review their own code first before submitting a PR; they do not expect others to catch sloppy mistakes for them.
+## Engineer II
 
-**Technical Scope:** Capable of taking well-scoped components from a larger project and completing these tasks in a reasonable time frame. Engineers at this level are assumed to be constantly making steady progress on tasks that are assigned to them and know when to ask for help when they are blocked. They can own their independent small-to-medium features all the way through from technical design to launch. An Engineer II is able to troubleshoot medium to high complexity bugs consistently, and execute the best available solution by assessing available options (quick fix, long term) against the urgency of the issue. An Engineer II does not require reminders or accountability coaching to remind them to write tests.
+An engineer II is capable of taking on most technical problems and coordinating work spanning several sprints.
 
-Engineer IIs are focused on learning about a specific component or product sub-area and mastering that component.
+**Technical Knowledge:** They have practical experience implementing the core concepts and best practices of software engineering. They are able to write performant, well-abstracted code with minimal help from other engineers. They are able to solve most problems in their domain, but may need help with complex tasks and tasks outside their domain.
 
-**Estimation:** Accountable and reliable to all commitments which they have taken ownership over, both in-office (by completing all work in the Sprint Backlog that they pulled in during Sprint Planning) and when On-Call (by promptly responding to incoming requests)
+**Productivity:** They reliably deliver the work assigned to them in a sprint. When they encounter a complex problem, they leverage the knowledge of their peers to find a suitable solution, even if it means changing the scope of the work. By the PR stage, their code is well-constructed; they do not expect others to catch sloppy mistakes for them.
```

Comment:
I would change the last sentence here "they do not expect others to catch sloppy mistakes for them". What if they don't expect it, but others are frequently catching sloppy mistakes in their PRs? I'd rather this just be oriented towards their PRs being largely "sloppy mistake free".